### PR TITLE
Update index.md to account for dumping changes

### PR DESCRIPTION
**Pull request by Eloeri**
With no longer being able to properly let yuzu derive keys due to changes in Hekate's dumping process, this proposed change overhauls the quickstart method. Added a section for dumping keys, and changed the process for injecting the payloads and dumping BIS Keys.

### CODE REVIEW

File: site/content/help/quickstart/index.md [hunk 1]
```diff
@@ -46,54 +46,56 @@ Sample Image:
...
     - 4e. Double-click on the row named `USER`. Enter the same key. Click `Mount`, and the tool may become unresponsive again. Once this mounts, open the new drive and copy the `Contents` and `save` folders to `%YUZU_DIR%/nand/user`. After, click `Unmount` and close the window and then the program.
 5. (OPTIONAL) If you have games or game data stored on your sd card, copy the `Nintendo` folder of your sd card into `%YUZU_DIR%/sdmc`.
-6. Open yuzu. You should see a dialog with a progress bar informing you that your keys are being derived. If you get a missing file error message, double check the files in your `sysdata` dir and try again. If there are no errors, a message informing you that it was successful will show and then yuzu will refresh to show you your games. 
+
+# Dump Prod.keys and Title.keys
+6. We will now dump your `prod.keys` and `title.keys` for decryption of your game files. Before you return your SD card to your Switch, place the `Lockpick.nro` you downloaded earlier from the requisties into the `switch` folder of your SD card.
```
Comment (Schplee) [inline site/content/help/quickstart/index.md]: `requisties` should be `prerequisites`

File: site/content/help/quickstart/index.md [hunk 2]
```diff
@@ -46,54 +46,56 @@ Sample Image:
...
+    - 6a. Reboot your Nintendo Switch into RCM mode, and reconnect it to your computer.
+    - 6b. Run TegraRCMGui, and in the `Payload` tab of TegraRCMGui, click on the folder icon and navigate to the `Lockpick_RCM` file you downloaded earlier from the prerequisites.
+    - 6c. There may be a message on your Switch to `Reboot to Sept...`, Press `Power` or `Vol +/-`
+    - 6d. After Lockpick_RCM has finished deriving the keys, please make note of the location fo the dump. Default is: `sd:/switch/prod.keys`.
```
Comment (Schplee) [inline site/content/help/quickstart/index.md]: `fo` should be `of`

File: site/content/help/quickstart/index.md (hunk 1 above)
Comment (Eloeri) [reply site/content/help/quickstart/index.md]: Thanks!

File: site/content/help/quickstart/index.md (hunk 2 above)
Comment (Eloeri) [reply site/content/help/quickstart/index.md]: Done, and done

**Pull request merged**